test/Microsoft.AspNetCore.Blazor.Build.Test/CodeGenerationTestBase.cs

```diff
@@ -2443,6 +2443,20 @@ public void OnComponentHover(UIMouseEventArgs e)
             CompileToAssembly(generated);
         }
 
+        [Fact]
+        public void EventHandlerTagHelper_EscapeQuotes()
+        {
+            // Act
+            var generated = CompileToCSharp(@"
+<input onfocus='alert(""Test"");' />
```

Comment:
I see that you have added a test for the following:
`<input onfocus='alert("Test");' />`

What about the swapping the single and double quotes around?  The following format would be my preferred way of writing the same input element:
`<input onfocus="alert('Test');" />`

Maybe worth adding another test to cover both formats if they are not already covered.

Reply:
I can also add a test case for that but it wouldn't serve much purpose since a single quote inside double quotes is valid c# code. Hence the single quotes don't need to be escaped at all.